p2panda-js/test/fixtures.ts

```diff
@@ -1,25 +1,51 @@
-import { marshallResponseFields } from '~/utils';
+import { OperationFields } from 'wasm';
 
-import TEST_DATA from './test-data.json';
+import { marshallResponseFields } from '~/utils';
 
 import type {
-  Entry,
-  Operation,
-  FieldsTagged,
   EncodedEntry,
+  Entry,
   EntryArgs,
+  FieldsTagged,
+  Operation,
+  OperationTagged,
+  OperationValue,
 } from '~/types';
 
+import TEST_DATA from './test-data.json';
+
 // Right now we only have one author `panda` who only has one schema log. This
 // could be expanded in the future.
-const PANDA_LOG = TEST_DATA.panda.logs[0];
+const encodedEntries = TEST_DATA.panda.logs[0].encodedEntries;
+const nextEntryArgs = TEST_DATA.panda.logs[0].nextEntryArgs;
+
+// Convert regular JavaScript object for operation fields into Map
+// @TODO: I'm horribly lost here in TypeScript land AAAAAAAAAAAAH!
+const decodedOperations = TEST_DATA.panda.logs[0].decodedOperations.map(
+  (operation): OperationTagged => {
+    if (operation.fields) {
+      const fields: FieldsTagged = new Map();
+
+      Object.keys(operation.fields).forEach((key: string) => {
+        const value: OperationValue = operation.fields[
+          'message'
+        ] as OperationValue;
+        fields.set(key, value);
+      });
+
+      operation.fields = fields;
+    }
+
+    return operation;
+  },
+);
```

Comment:
I have no clue on how to do this in TypeScript (convert from JSON into different type), might need some help here :cry: 

@cafca